hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -446,6 +446,22 @@
       swaccess: "rw",
       hwaccess: "hro",
       fields: [
+        { bits: "0",
+          name: "FLASH_STATUS_FIFO_CLR",
+          swaccess: "rw1s",
```

Comment:
if you wanted to, you could also encode this as `r0w1c`, and just rely on the q, qe bits (I think you wouldn't have to implement the automatic clear to 0 in that case).

Reply:
Haha, I just didn't know this was an option. How useful!

Reply:
Hm... it doesn't look like that produces a result where we get to see the `1` on the `q` or `ds` outputs. The `w1c` part causes SW to always write 0 to the flop.